src/mod/event_handlers/mod_event_socket/mod_event_socket.c

```diff
@@ -1109,13 +1109,15 @@ SWITCH_STANDARD_API(event_sink_function)
 					switch_xml_free(xml);
 				} else {
 					stream->write_function(stream, "<data><reply type=\"error\">XML Render Error</reply></data>\n");
-					break;
+					goto endloop;
```

Comment:
That continues the loop, does not break. Why?

Reply:
sorry i'm wrong

i dont see at  https://github.com/signalwire/freeswitch/blob/da8703b2151ebdbb9331a10c5814c016cdd282c4/src/mod/event_handlers/mod_event_socket/mod_event_socket.c#L1137

thanks for your time